testnets/atomonetestnet/versions.json

```diff
@@ -2,6 +2,31 @@
   "$schema": "../../versions.schema.json",
   "chain_name": "atomonetestnet",
   "versions": [
+    {
+      "name": "v2.0.0-rc2",
```

Comment:
Please place this latest version at the bottom of the array

Reply:
fixed here [a1ff701](https://github.com/cosmos/chain-registry/pull/6271/commits/a1ff70166b3a436d7c5675df1d20d0c52f4a86f4)